spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -241,17 +232,31 @@ private Expression conflictDetectionFilter(SparkBatchQueryScan queryScan) {
 
     @Override
     public void abort(WriterCommitMessage[] messages) {
-      if (!cleanupOnAbort) {
-        return;
+      if (cleanupOnAbort) {
+        SparkCleanupUtil.deletePaths("job abort", table.io(), filePaths(messages));
+      } else {
+        LOG.warn("Skipping cleanup of written files, unable to determine the final commit state");
```

Comment:
The "skipping cleanup of written files" part makes sense to me, but wouldn't "unable to determine the final commit state" apply for both cases (any abort case)?  Or are we trying to indicate that we won't be cleaning up any orphan files

Reply:
I adapted the original comment but I agree it is a bit weird as the var name is generic and does not say anything about commit state. I changed.